src/Compatibility/Microsoft.DotNet.ApiCompatibility/Microsoft.DotNet.ApiCompatibility.csproj

```diff
@@ -3,15 +3,16 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <StrongNameKeyId>Open</StrongNameKeyId>
-    <VersionPrefix>1.0.0</VersionPrefix>
     <!-- We need to compare ISymbols in a special way (by name) and roslyn symbol comparers take more heuristics into consideration.-->
     <NoWarn>RS1024;$(NoWarn)</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- ExcludeAssets="Runtime" is being set to avoid adding package references and their dependencies into the package. -->
     <!-- We pin the versions of the assets that ship with the SDK in order to be able to run on older SDKs. 
     Currently we support .NET 6 Preview5 or later. -->
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="4.0.0-1.21304.5" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="4.0.0-1.21304.5"  ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" ExcludeAssets="Runtime" />
```

Comment:
Why are we adding `Microsoft.Build.Framework` here? `API Compat` shouldn't depend on any MSBuild things at the moment.